adopters.geojson

```diff
@@ -88,5 +88,7 @@
     {"type": "Feature", "properties": {"institution": "RIADVICE"}, "geometry": {"type": "Point", "coordinates": [10.2057799, 36.7088573]}},
     {"type": "Feature", "properties": {"institution": "PrivacyLx"}, "geometry": {"type": "Point", "coordinates": [-9.15808, 38.74410]}},
     {"type": "Feature", "properties": {"institution": "Ventspils University of Applied Sciences"}, "geometry": {"type": "Point", "coordinates": [21.5461, 57.3848]}}
+    {"type": "Feature", "properties": {"institution": "i3Campus - Colombia"}, "geometry": {"type": "Point", "coordinates": [4.67537,-74.12198]}}
+    
```

Comment:
Thanks for adding your installation!
Could you please just add this suggestion, to format the json correctly:
```suggestion
    {"type": "Feature", "properties": {"institution": "Ventspils University of Applied Sciences"}, "geometry": {"type": "Point", "coordinates": [21.5461, 57.3848]}},
    {"type": "Feature", "properties": {"institution": "i3Campus - Colombia"}, "geometry": {"type": "Point", "coordinates": [4.67537,-74.12198]}}
```